awacs/a2c.py

```diff
@@ -5,13 +5,14 @@
 
 from .aws import Action as BaseAction
 from .aws import BaseARN
+from typing import Optional
```

Comment:
Imports from the stdlib should go to the top.  `isort awacs/` should fix that.  I'd say run `black awacs/` as well (to enforce the max. line length), just in case.

Reply:
Ahhh, good point. I ran isort/black and pushed a new version. Thank you for the review.